pkg/chunkserver/config.go

```diff
@@ -1,9 +1,15 @@
 package chunkserver
 
-import "strconv"
+import (
+	"strconv"
+
+	"github.com/opencurve/curve-operator/pkg/config"
+)
 
-// chunkserverConfig for a single chunkserver
 // chunkserverConfig implements config.ConfigInterface
+var _ config.ConfigInterface = &chunkserverConfig{}
```

Comment:
init func() maybe better?

Reply:
IMO

Reply:
Not necessary. It's only a check and not used anywhere.

Reply:
And that's why I use _ as the variable name.

Reply:
ok, make successed? because ci system is not added.

Reply:
If the struct doesn't implement the interface, go build will fail, so does `make run`.

We can add CI later.

Reply:
If you are willing to together so much the better^^